client/model.go

```diff
@@ -43,3 +47,38 @@ func (client *Client) Remodel(b []byte) (changeID string, err error) {
 
 	return client.doAsync("POST", "/v2/model", nil, headers, bytes.NewReader(data))
 }
+
+// CurrentModelAssertion returns the current model assertion
+func (client *Client) CurrentModelAssertion() (asserts.Assertion, error) {
+	return currentAssertion(client, "/v2/model")
+}
```

Comment:
this should return the actual final type  *asserts.Model

Reply:
done